forui/lib/src/theme/theme.dart

```diff
@@ -3,13 +3,10 @@ import 'package:flutter/material.dart';
 
 import 'package:forui/forui.dart';
 
-/// Represents a ForUI theme.
+/// Represents a Forui theme.
+///
+/// See [ThemeBuildContext.theme] for accessing the current theme.
 class FTheme extends StatelessWidget {
-  /// Retrieves the theme data.
-  static FThemeData of(BuildContext context) {
-    final theme = context.dependOnInheritedWidgetOfExactType<_InheritedTheme>();
-    return theme?.data ?? FThemes.zinc.light;
-  }
```

Comment:
I think this should stay. It's a pretty standard pattern to include the `of` method.